src/details-element-polyfill/polyfill.coffee.erb

```diff
@@ -35,6 +35,8 @@ polyfillProperties = ->
 
   Object.defineProperties prototype,
     open:
+      get: ->
+        @hasAttribute("open")
```

Comment:
Since `prototype` might `HTMLUnknownElement` (instead of `HTMLDetailsElement`) when polyfilling, we need to  check if `tagName` is `DETAILS` and make sure to call the original `get`er. Juist like the `set`er does.

Reply:
Done. Thanks! ⭐